general/server/plugins/addons.md

```diff
@@ -0,0 +1,85 @@
+---
+uid: server-plugins-Addons
+title: Addons
+---
+
+
+### ErsatzTV
+
+#### Description
+ErsatzTV is an amazing Jellyfin addon that is a great way to take your Jellyfin libraries and make them into Live TV Channels. For example if i wanted to link All episodes of Trailer Park Boys and shuffle them at random like you were watching it on live tv this plugins can do it. Not only that but you can make custom channel names, Logos, Watermarks transcodings and add Movies, TV-Shows, and Music Collections of all sorts. This is a must if you want a cable like experience without paying or having buggy iptv interfaces.
+
+**Link:**
+
+* [Github](https://github.com/jasongdove/ErsatzTV)
+
+### JF-Secure
+
+#### Description
+A turn key Caddy Reverse Proxy Webserver setup for Jellyfin without any codiung required.
```

Comment:
```suggestion
A  turn-key reverse proxy tool that makes use of Caddy. It will setup a reverse proxy for Jellyfin with the goal of being easy to use, even for novice users.
```

I do not know the project deeply as this is the first time that I heard it. But what do you think of this description?

Reply:
okay i made all the grammar corrections thank you for the advice it does sound and look a lot better.